abstract.tex

```diff
@@ -52,22 +52,23 @@
 \begin{center}
 \textbf{Resum\'e}
 \end{center}
-In this thesis, I describe two things I did:
+Foreliggende afhandling redegør for to af mine bedrifter:
 \begin{enumerate*}[label=\arabic*)]
-\item a compiler transformation that
-implements performant automatic differentiation (AD) for a functional,
-data-parallel language and
-\item an approach for rank polymorphism in a statically typed language with
-  parametric polymorphism and type inference.
+\item udførsel af en programtransformation indbygget i en oversætter,
+  der kan udføre automatisk differentiering (AD) af programmer skrevet
+  i et funktionsorienteret og dataparallelt programmeringssprog, samt
+\item en tilgang til rangpolymorfi i et programmeringssprog med
+  statiske typer, parametrisk polymorfi, og typeinferens.
```

Comment:
I don' think the last comma is correct.

Reply:
That follows from [§46.1](https://ro.dsn.dk/?type=rulesearch&side=46&paragraf=1).

Reply:
Both you and DSN are wrong.